share/completions/systemd-analyze.fish

```diff
@@ -16,9 +16,18 @@ complete -c systemd-analyze -l to-pattern -d 'dot: show relationships matching r
 complete -c systemd-analyze -l fuzz -x -d 'critical-chain: also show units which finished timespan earlier than last unit in same level'
 complete -c systemd-analyze -l man -xa no -d 'Do not invoke man to verify the existence of man pages'
 complete -c systemd-analyze -l generators -d 'Invoke unit generators'
+complete -c systemd-analyze -l instance -r -d 'Fallback instance name for template units'
 complete -c systemd-analyze -l root -xa "(__fish_complete_directories)" -d 'With cat-files, show config files underneath the specified root path'
+complete -c systemd-analyze -l image -xa "(__fish_complete_path)" -d 'With cat-files, show config files inside the specified image path'
```

Comment:
__fish_complete_path completes executable files which we don't want here.
Replace `-xa "(__fish_complete_path)"` with `-r` to complete file paths.